apps/token/src/routes/rewards/epoch-total-rewards/epoch-total-rewards-table.tsx

```diff
@@ -22,191 +17,46 @@ interface RewardItemProps {
   last?: boolean;
 }
 
-const displayReward = (reward: string) => {
-  if (Number(reward) === 0) {
-    return <span className="text-vega-dark-300">0</span>;
-  }
+const DisplayReward = (reward: string) => {
+  const {
+    appState: { decimals },
+  } = useAppState();
 
-  if (reward.split('.')[1] && reward.split('.')[1].length > 4) {
-    return (
-      <Tooltip description={formatNumber(reward)}>
-        <button>{formatNumber(Number(reward).toFixed(4))}</button>
-      </Tooltip>
-    );
+  if (Number(reward) === 0) {
+    return <span className="text-vega-dark-300">-</span>;
   }
 
-  return <span>{formatNumber(reward)}</span>;
+  return (
+    <Tooltip description={formatNumber(toBigNum(reward, decimals), decimals)}>
+      <button>{formatNumber(toBigNum(reward, decimals))}</button>
+    </Tooltip>
+  );
 };
 
-const gridStyles = classNames(
-  'grid grid-cols-[repeat(8,minmax(100px,auto))] max-w-full overflow-auto',
-  `border-t border-vega-dark-200`,
-  'text-sm'
-);
-
-const headerGridItemStyles = (last = false) =>
-  classNames('border-r border-b border-b-vega-dark-200', 'py-3 px-5', {
-    'border-r-vega-dark-150': !last,
-    'border-r-0': last,
-  });
-
-const rowGridItemStyles = (last = false) =>
-  classNames('relative', 'border-r border-b border-b-vega-dark-150', {
-    'border-r-vega-dark-150': !last,
-    'border-r-0': last,
-  });
-
-const ColumnHeader = ({
-  title,
-  tooltipContent,
-  className,
-}: ColumnHeaderProps) => (
-  <div className={className}>
-    <h2 className="mb-1 text-sm text-vega-dark-300">{title}</h2>
-    {tooltipContent && (
-      <Tooltip description={tooltipContent}>
-        <button>
-          <Icon name={'info-sign'} className="text-vega-dark-200" />
-        </button>
-      </Tooltip>
-    )}
-  </div>
-);
-
 const RewardItem = ({ value, dataTestId, last }: RewardItemProps) => (
   <div data-testid={dataTestId} className={rowGridItemStyles(last)}>
     <div className="h-full w-5 absolute right-0 top-0 bg-gradient-to-r from-transparent to-black pointer-events-none" />
-    <div className="overflow-auto p-5">{displayReward(value)}</div>
+    <div className="overflow-auto p-5">{DisplayReward(value)}</div>
```

Comment:
suggestion (non-blocking): same here with the function vs using is a react component like `<DisplayReward value={value} />`. not sure, maybe if this is a pattern more commonly used in the code here, then probably not that important